packages/clay-card/stories/index.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import CardFile from '../fixtures/file/CardFile';
+import CardFileSelectable from '../fixtures/file/CardFileSelectable';
+
+import CardFolder from '../fixtures/folder/CardFolder';
+import CardFolderSelectable from '../fixtures/folder/CardFolderSelectable';
+
+import CardHorizontalInteractive from '../fixtures/interactive/CardHorizontalInteractive';
+
+import CardImage from '../fixtures/image/CardImage';
+import CardImageSelectable from '../fixtures/image/CardImageSelectable';
+
+import CardInteractive from '../fixtures/interactive/CardInteractive';
+import CardInteractiveNoTruncateDescription from '../fixtures/interactive/CardInteractiveNoTruncateDescription';
+import CardInteractiveWithIcon from '../fixtures/interactive/CardInteractiveWithIcon';
+
+import CardUser from '../fixtures/user/CardUser';
+import CardUserSelectable from '../fixtures/user/CardUserSelectable';
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
```

Comment:
We do not need to add spaces between the imports.

Reply:
Solved